src/main/webapp/index.html

```diff
@@ -1,15 +1,17 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
       <meta charset="utf-8" />
-      <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, minimal-ui">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
       <title>noms</title>
+      <link rel="icon" href="./assets/favicon.ico" type="image/x-icon">
```

Comment:
Would it be more clear to be bento-box-icon instead of x-icon?

Reply:
`x-icon` isn't an arbitrary descriptor, but the [MIME type](https://stackoverflow.com/questions/13827325/correct-mime-type-for-favicon-ico) of the favicon :0 Replacing it with something arbitrary would be the same as replacing `type="text/javascript"` with `type="text/name-of-javascript-file"`, which would cause the resource to be served incorrectly